src/index.js

```diff
@@ -1,13 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App/App';
+import './CSS/index.scss';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import App from './App/Components/App';
 import reportWebVitals from './reportWebVitals';
+import AllReducers from './Redux/reducers/index';
 // import * as serviceWorkerRegistration from './serviceWorkerRegistration';
 
+const store = createStore(AllReducers);
```

Comment:
- [ ] [REQUIRED] it adds example books to initial state object

Reply:
@sergiocortessat let's work on these changes.